### PR TITLE
Rework DB structure

### PR DESCRIPTION
**Pull request by max8rr8**

### CODE REVIEW

File: db/fakeData.sql [hunk 1]
```diff
@@ -20,7 +20,7 @@ VALUES ('test user 3',
         'fc4bda146f520912cf0cadd32e3ed541'); -- Password testtest3
 
 
-INSERT INTO magnet_types(name, main_color, second_color, icon, created_by)
+INSERT INTO magnets(name, main_color, second_color, icon, created_by)
```
Comment (Pavel-Demidyuk) [inline db/fakeData.sql]: **created_by** rename to **author**

File: db/migrations/1.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+\c e_magnet
+ALTER TABLE magnets RENAME TO users_magnets;
+ALTER TABLE users_magnets RENAME COLUMN magnet_id TO user_magnet_id;
```
Comment (Pavel-Demidyuk) [inline db/migrations/1.sql:3]: **user_magnet_id;** why not just **user_id** ?

**Review by Pavel-Demidyuk — CHANGES_REQUESTED** [review]

File: db/migrations/1.sql (hunk 2 above)
Comment (max8rr8) [reply db/migrations/1.sql:3]: because it is id of relationship

**Comment by Pavel-Demidyuk** [comment]
Добавляй описание в каждый пулл реквест

**Comment by Pavel-Demidyuk** [comment]
go ahead and merge, great job!

**Pull request merged**